### PR TITLE
RHDEVDOCS-3850 Add link to solution: How to use builds with Red Hat S…

### PR DESCRIPTION
**Pull request by rolfedh**
Aligned team: Dev Tools
- For branches: 4.10+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3850
- Direct link to doc preview: Scroll down to first "Additional resources" section below https://deploy-preview-43015--osdocs.netlify.app/openshift-enterprise/latest/cicd/builds/running-entitled-builds.html#running-builds-with-red-hat-satellite-subscriptions
- SME review: Not needed 
- QE review: Not needed
- Peer review: @Srivaralakshmi

### CODE REVIEW

File: cicd/builds/running-entitled-builds.adoc [hunk 1]
```diff
@@ -22,6 +22,10 @@ include::modules/builds-source-input-satellite-config.adoc[leveloffset=+2]
 
 include::modules/builds-strategy-docker-entitled-satellite.adoc[leveloffset=+2]
 
+.Additional resources
```
Comment (Srivaralakshmi) [inline cicd/builds/running-entitled-builds.adoc:25]: Suggestion: 

- Please add [role="_abstract"] before L9.
- I see that the [OCP guidelines for additional resources](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) mention this: "the . formatting of the include statement indicates that the resource applies specifically to the module and not to the assembly. For example:

[role="_additional-resources"]
.Additional resources
* link:some-url.com[Human readable label]
* xref:some_xref[Human readable label]
* xref:some_other_xref[Human readable label]"

 Based on this guideline, I believe you should add _[role="_additional-resources"]_ before L25. PTAL and do the needful.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@rolfedh Left a couple of suggestions. PTAL. Thanks!

File: cicd/builds/running-entitled-builds.adoc (hunk 1 above)
Comment (rolfedh) [reply cicd/builds/running-entitled-builds.adoc:26]: That's correct. Updated. Thank you.

**Review by Srivaralakshmi — APPROVED** [review]
LGTM

**Comment by bergerhoffer** [comment]
@Preeticp @rolfedh this PR was labeled for 4.9 but not cherry-picked to enterprise-4.9. Just checking if it needed to be. Thanks!

**Comment by rolfedh** [comment]
> @Preeticp @rolfedh this PR was labeled for 4.9 but not cherry-picked to enterprise-4.9. Just checking if it needed to be. Thanks!

Thanks for catching that, Andrea. That "4.9+" is an error. This is for "4.10+" only. This PR doesn't need to be cherry-picked to 4.9. I'll update the description now.

**Comment by Preeticp** [comment]
Thanks Andrea and Rolfe!

**Comment by bergerhoffer** [comment]
Thanks @rolfedh! I'll remove the enterprise-4.9 label as well.

**Pull request merged**